lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -2828,27 +2842,15 @@ LogicalResult FIRRTLLowering::visitDecl(NodeOp op) {
   // Node operations are logical noops, but may carry annotations or be
   // referred to through an inner name. If a don't touch is present, ensure
   // that we have a symbol name so we can keep the node as a wire.
-  auto symName = getInnerSymName(op);
   auto name = op.getNameAttr();
-  if (AnnotationSet::removeAnnotations(op, dontTouchAnnoClass) && !symName) {
```

Comment:
Not 100% but maaaaaybe it matters that we remove the annotation, re:the warnings we emit for unhandled annotations? Sorry if that's addressed, going through this linearly :+1: .

Reply:
Yeah I think your'e right, going to have to delete the annotation.

Reply:
I made the function "getLoweredSymbol" drop the annotation. Its seems kind ugly for that function to have a side effect on the operation, but it was expedient.